src/components/ArticleCard.jsx

```diff
@@ -0,0 +1,35 @@
+import { useParams } from "react-router-dom";
+import { useEffect, useState } from "react";
+import {getArticle} from "../api";
+
+
+
+const ArticleCard = () => {
+    
+    const {article_id} = useParams();
+    const [article, setArticle] = useState([])
+
+    useEffect(() => {
+        getArticle(article_id).then((article) => {
+            setArticle(article)
+
+        })
+    }, [article_id])
+
+
+    return (
+        <main>
+            <h2> {article.title}</h2>
+             <span> By {article.author}</span> <br />
```

Comment:
Span doesn't have much semantic meening, is there another tag you could use or wrap outside of this that would be better to signify the contents